src/web/components/AdSlot.tsx

```diff
@@ -174,7 +214,7 @@ export const AdSlot: React.FC<Props> = ({ position, display }) => {
 								'ad-slot--rendered',
 								'js-sticky-mpu',
 							].join(' ')}
-							css={labelStyles}
+							css={[labelStyles, fluidAdStyles]}
```

Comment:
If we apply `labelStyles` and `fluidStyles` in all cases could we combine them as:

```js
const adStyles = { ...labelStyles, ...fluidStyles }
```

Reply:
I was thinking about this, hence why I wanted to confirm which ads require 100% width with DAP.